lib/services/thinkingParser.js

```diff
@@ -245,14 +293,14 @@ function blackButtonRequestCreation(fields, result, timestamp, callback) {
         metadatas: timestamp
     });
     result.attributes.push({
-        name: 'op_status',
-        value: 'PENDING',
+        name: 'last_operation',
+        value: result.operation,
         type: 'string',
         metadatas: timestamp
     });
     result.attributes.push({
-        name: 'last_operation',
-        value: result.operation,
+        name: 'op_status',
+        value: 'PENDING',
```

Comment:
We do, but in the current scenario we are using the old ones. We will consider the use of constants then. NTC